### ISSUE TITLE
【accuracy】I trained your modle ，when the epoch is 350，the accuracy is 56.45%、41.81%、33.51% by CRNN

### ISSUE THREAD

**Issue by Lz-2019317** [FudanVI/FudanOCR#11]
is that correct？
The paper results is 59.6%、47.1%、35.3% by CRNN,could you describe your trianed detail？ that will give me tremendous help！
Thank you！

**Comment by JingyeChen** [comment]
It is not correct. You could refer to the pre-trained model and log file.
By the way,
What is the version of your PyTorch and TorchVision?
What is the batch size?

**Comment by Lz-2019317** [comment]
hello&nbsp;
my pytorch version=1.8
batchsize =80，（the paper said）
and i tried to train the model，which batchsize=12
thank you







------------------ Original ------------------
From: JingyeChen ***@***.***&gt;
Date: Wed,Aug 25,2021 3:04 PM
To: FudanVI/FudanOCR ***@***.***&gt;
Cc: Zhen Liu ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [FudanVI/FudanOCR] 【accuracy】I trained your modle ，when the epoch is 350，the accuracy is 56.45%、41.81%、33.51% by CRNN (#11)





 
It is not correct. You could refer to the pre-trained model and log file.
 By the way,
 What is the version of your PyTorch and TorchVision?
 What is the batch size?
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by JingyeChen** [comment]
You need to follow the file "requirement.txt" to set up the environment.
The PyTorch version should be 1.2.0
(I also observe that the performance will drop when using PyTorch of higher versions)

Furthermore, the performance will be a little bit better when the batch size is set to 16 :D

**Comment by JingyeChen** [comment]
Furthermore, please note that torchvision==0.2.1

**Comment by Lz-2019317** [comment]
thank you very much ！
i will try again！
your reply give a lot of help ！
thank you！



发自我的iPhone


------------------ Original ------------------
From: JingyeChen ***@***.***&gt;
Date: Wed,Aug 25,2021 3:52 PM
To: FudanVI/FudanOCR ***@***.***&gt;
Cc: Zhen Liu ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [FudanVI/FudanOCR] 【accuracy】I trained your modle ，when the epoch is 350，the accuracy is 56.45%、41.81%、33.51% by CRNN (#11)





 
You need to follow the file "requirement.txt" to set up the environment.
 The PyTorch version should be 1.2.0
 (I also observe that the performance will drop when using PyTorch of higher versions)
 
Furthermore, the performance will be a little bit better when the batch size is set to 16 :D
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by JingyeChen** [comment]
没事:D，老哥你也是国内的吧，中文交流方便一点

**Comment by Lz-2019317** [comment]
哈哈哈哈哈，那就好，感谢感谢



发自我的iPhone


------------------ Original ------------------
From: JingyeChen ***@***.***&gt;
Date: Wed,Aug 25,2021 3:57 PM
To: FudanVI/FudanOCR ***@***.***&gt;
Cc: Zhen Liu ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [FudanVI/FudanOCR] 【accuracy】I trained your modle ，when the epoch is 350，the accuracy is 56.45%、41.81%、33.51% by CRNN (#11)





 
没事:D，老哥你也是国内的吧，中文交流方便一点
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Lz-2019317** [comment]
大佬，再打扰一下您！
1.pytorch版本影响差别大吗？有1个百分点吗？
2.学习率有没有试过设置成0.0002？
3.我batchsize设置为54是不是太大？





发自我的iPhone


------------------ Original ------------------
From: JingyeChen ***@***.***&gt;
Date: Wed,Aug 25,2021 3:57 PM
To: FudanVI/FudanOCR ***@***.***&gt;
Cc: Zhen Liu ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [FudanVI/FudanOCR] 【accuracy】I trained your modle ，when the epoch is 350，the accuracy is 56.45%、41.81%、33.51% by CRNN (#11)





 
没事:D，老哥你也是国内的吧，中文交流方便一点
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by JingyeChen** [comment]
3.batchsize最好设低一点，如果想取得更高性能的话，建议设为16
前两个问题我这边没有记录实验数据，您可以尝试一下哈

**Comment by Lz-2019317** [comment]
大神，文中的提到self-sttention机制，可以关联特征图中的任意像素对，从而对处理任意方向的文本图像具有鲁棒性。也就是说，self-attention机制主要关注位置的信息 ，如果再加入通道或者空间注意力模块，会不会有冗余或者冲突呢？

**Comment by JingyeChen** [comment]
我个人觉得不会冲突

**Comment by Lz-2019317** [comment]
谢谢~~训练好权重以后，演示demo中放入的图片尺寸也要跟原来数据集一样才可以吗？

**Comment by JingyeChen** [comment]
可以不一样

**Comment by linminhtoo** [comment]
Hi @JingyeChen  can you elaborate why the performance will drop at higher torch versions? I want to change the augmentation to synthesise LR from HR to GaussianBlur, and this is only available on torch 1.7.0. I could use other libraries, but it adds extra packages
